src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+However, Hegel detects a more serious obstacle in this thinking when it is attempted to be philosophically applied. When, according to this naive position, thoughts are _only_ thoughts and that real knowledge is the intertwining of thoughtful form and worldly impressions, it can be easily missed that what are assumed to be conflicts of the world are really conflicts of ideas. In Hegel's more technical vocabulary, it is _the conflict of determinations_.
+
+> The basis of the [conception of modern philosophy is] in the insight into the _necessary conflict_ of the determinations of the understanding with themselves. - The reflection already mentioned consists in _transcending_ the concrete immediate, in determining_ and _parting_ it. But this reflection must _equally transcend_ its _separating_ determinations and above all _connect_ them. The conflict of determinations breaks out precisely at the point of connection. This reflective activity of connection belongs in itself to reason, and to rise above the determinations ad attain insight into their discord is the great negative step on the way to the true concept of reason (Hegel, 2010, 26/21.30).
+
+When logic is not considered on its own terms but merely in relation to sensuous existence, one fails to grasp that logic is in fact in conflict with itself. Discord and struggle are not exclusive to the domain of worldly reality but are also&mdash;perhaps more intensely&mdash;active in the domain of logic and reason. Philosophy disciplines the ability to distinguish what is proper to each domain and being able to understand when an apparent conflict belongs to worldly reality, the realm of pure logic or is an erroneous mismatch between the two.
+
+Hegel notes a further error in the case where one is discouraged by a contradiction in the understanding and resolves to find accord and stability in sensuous reality. But, and here Hegel criticizes the Kantian transcendental standpoint once more, if one resolves to look to sensuous reality as the criterion of truth, in the same instance one cannot determine the thing in question to have _any_ independent truth since the truth of the matter is settled _only_ as it appears. "This is like attributing right insight to someone," Hegel writes, "with the stipulation, however, that he is not fit to see what is true but only what is false" (Hegel, 2010, 26/21.30). To put this another way, if by truth one understands a independent standard, then this immediately clashes with the knowledge through appearances, since knowing here is totally dependent on appearances and cannot make _any_ independent claims about the matter at hand. To emphasize this point further, if the transcendental arrangement is actually granted, then one cannot truly speak of laws of nature or even state that the sky is there the moment one turns one's gaze towards the ground. This is why, if Hegel is right, this claim that sensuous reality is the sole arbiter of truth leads to absurdity.
```

Comment:
" with the knowledge through appearances" - I think this would be clearer if it said " with the knowledge that one acquires through appearances", if I've understood what you're going for.

Reply:
Yep, better. Added